lib/chewy/type/import.rb

```diff
@@ -126,6 +126,7 @@ def compose(object, crutches = nil, fields: [])
       private
 
         def import_routine(*args)
+          return if args.first.blank? && !args.first.nil?
```

Comment:
What if only the options are passed?
It probably makes sense to do this check after `args.extract_options!`

Reply:
In that case, given that options aren't empty, this would work the same as `Index.import!` (without objects, with imports everything under the scope). Right?

Reply:
I don't mind changing this to make it less confusing though.